sklearn/decomposition/tests/test_sparse_pca.py

```diff
@@ -86,6 +90,38 @@ def test_fit_transform(norm_comp):
                          Y, ridge_alpha=None)
 
 
+@pytest.mark.filterwarnings("ignore:normalize_components")
+@pytest.mark.parametrize("norm_comp", [False, True])
+def test_fit_transform_variance(norm_comp):
+    '''
+        This function asserts that the variance computed by SparsePCA is the
```

Comment:
We use comments rather than docstrings in tests (although mostly for historical reasons)